### PR TITLE
Fixed the buggy click behaviors

### PR DESCRIPTION
**Pull request by kanastasakis26**
Try it out for yourself.

### CODE REVIEW

File: client/ts/graph_chart.ts [hunk 1]
```diff
@@ -22,19 +25,32 @@ export function generateGraphChart(jsonUrl){
...
+            .attr('stroke-opacity', 0.0)
+            .on('click', onLinkClick);
+
+
         let links = svg.append("g")
             .selectAll("line")
             .data(data.links)
             .enter()
             .append("line")
+            .attr("pointer-events", "none")
```
Comment (xyz8983) [inline client/ts/graph_chart.ts:48]: fancy attr

File: client/ts/graph_chart.ts [hunk 2]
```diff
@@ -22,19 +25,32 @@ export function generateGraphChart(jsonUrl){
         // .force('collide', d3.forceCollide().radius(nodeRadius*1.2))
         .force('collide', d3.forceCollide())
         .force('link', d3.forceLink().distance(80)) // distance sets the length of each link
-        .force('link', d3.forceLink().id(function(d){ return d.name; }));  // for source and targe value in links not using number based zero but based on customized string
+        .force('link', d3.forceLink().id(function(d:any){ return d.name; }));  // for source and targe value in links not using number based zero but based on customized string
 
     d3.json(jsonUrl.value, function(error, data){
-        console.log("data",data);
+        // These thicker lines make it easier for someone to click on a graph edge.
+        let bg_links = svg.append("g")
```
Comment (xyz8983) [inline client/ts/graph_chart.ts:32]: could you explain the idea of having two links?

Comment (kanastasakis26) [reply client/ts/graph_chart.ts:32]: There are two layers of links. One is wider than the other. The wider line is there to increase the click area of the edge. It is also transparent. The thinner one is the one you see. It does not respond to mouse events.

File: client/ts/graph_chart.ts (hunk 1 above)
Comment (kanastasakis26) [reply client/ts/graph_chart.ts:48]: It turns off mouse events.

**Pull request merged**